web/src/app/administration/classification/details/classification-details.component.ts

```diff
@@ -224,18 +222,17 @@ export class ClassificationDetailsComponent implements OnInit, OnDestroy {
   }
 
   private async selectClassification(id: string) {
-    if (this.classificationIsAlreadySelected()) {
-      return true;
+    if (!this.classificationIsAlreadySelected()) {
+      this.requestInProgress = true;
+      const classification = await this.classificationsService.getClassification(id);
+      this.fillClassificationInformation(classification);
+      this.classificationsService.selectClassification(classification);
+      this.requestInProgress = false;
```

Comment:
return true is missing? not  needed?

Reply:
No, because the return value is never catched.